src/components/commons/inputs/Dropdown.tsx

```diff
@@ -33,18 +33,16 @@ export const Dropdown = ({
   options,
   isSmall = false,
   isDisabled = false,
-  setState = undefined,
+  onChange,
 }: DropdownProps) => {
   const { register, setValue } = useFormContext();
   const { isOpen, popupRef, buttonRef, togglePopup } = useTogglePopup();
-  const [currentOption, setCurrentOption] = useState(typeof options[0] === 'object' ? options[0].title : options[0]);
+  const [currentOptionTitle, setCurrentOptionTitle] = useState(options[0]?.title || '');
```

Comment:
코드의 56번째 줄에서는 `options[0]?.value` 가 아닌  `options[0].value` 로 쓰이고 있는데,

만약 options배열이 반드시 첫번째 요소를 가진다면 여기서 그냥 아래처럼 써도 될거 같습니다!
```suggestion
  const [currentOptionTitle, setCurrentOptionTitle] = useState(options[0].title);
```

하지만 첫번째 요소가 없을수도 있는 거라면 56번 줄에 `options[0]?.value ?? 0` 이런 식으로 null 처리를 해줘야 할 것 같습니다!

Reply:
궁금한 점이 있습니다! options 배열은 저희가 필수로 넘겨주는거 아닌가요? 옵셔널체이닝 쓰는 이유가 궁금합니다.